scripts/ast-patches/allowAnyPropOnPlayableMixinNext.js

```diff
@@ -0,0 +1,27 @@
+const ts = require("typescript");
+
+function allowAnyPropOnPlayableMixinNext(ast) {
+  const $w = ast.statements.find(
+    statement =>
+      ts.isModuleDeclaration(statement) && statement.name.escapedText === "$w"
+  );
+  const playableMixin = $w.body.statements.find(
+    statement =>
+      ts.isInterfaceDeclaration(statement) &&
+      statement.name.escapedText === "PlayableMixin"
```

Comment:
I would extract all hardcoded string to one consts file

Reply:
i don't think it'll give any value here... it's only used once